packages/core/src/accounts/serializers.ts

```diff
@@ -28,6 +29,7 @@ export function serializeAccount({
     spendableBalance: spendableBalance.toString(),
     blockHeight,
     lastSyncDate: lastSyncDate.toISOString(),
+    parentAccountId,
```

Comment:
I think you also need the same in the `deserializeAccount` function

Reply:
Done, good catch